lib/neo4j/migrations/helpers.rb

```diff
@@ -0,0 +1,50 @@
+module Neo4j
+  module Migrations
+    module Helpers
+      def rename_property(label, old_property, new_property)
+        by_label(label).where(n: {new_property => nil})
+                       .set("n.#{new_property} = n.#{old_property}")
+                       .remove("n.#{old_property}")
+                       .exec
+      end
+
+      def drop_nodes(label)
+        query.match("(n:`#{label}`)")
+             .optional_match('(n)-[r]-()')
+             .delete(:r, :n).exec
+      end
+
+      def add_labels(label, *labels)
+        by_label(label).set("n:#{labels.join(':')}").exec
+      end
+
+      def remove_labels(label, *labels)
+        by_label(label).remove("n:#{labels.join(':')}").exec
+      end
+
+      alias add_label add_labels
+      alias remove_label remove_labels
+
+      def remove_constraint(label, property)
+        execute("DROP CONSTRAINT ON (n:`#{label}`) ASSERT n.#{property} IS UNIQUE")
```

Comment:
I believe that you can do something like `Neo4j::Label.new(label). create_constraint(property, type: :unique)`.  @subvertallchris ?